ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>Vertex Pipelines -&gt; Vertex AI Pipelines</p><p> </p><p>you'll -&gt; you (TW guidelines is to use present tense, not future tense).</p><p> </p><p>Drop "The basics of" (TW guideline with bullets is to start with a verb when possible).</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>According to that notebook of renaming, and also the docs (https://cloud.devsite.corp.google.com/vertex-ai/docs/pipelines), it is "Vertex Pipelines"</p>